src/main/java/org/wildfly/client/config/ClientConfiguration.java

```diff
@@ -259,20 +264,25 @@ public static ClientConfiguration getInstance(ClassLoader classLoader) {
     }
 
     static URI propertyUrlToUri(String wildFlyConfig) {
-        try {
-            URI uri = new URI(wildFlyConfig);
-            if (! uri.isAbsolute()) { // URI does not include schema
-                if (uri.getPath().charAt(0) != File.separatorChar && uri.getPath().charAt(0) != '/') { // relative path
-                    String userDir = System.getProperty("user.dir").replace(File.separatorChar, '/');
-                    return Paths.get(userDir, uri.getPath()).toUri();
-                } else { // absolute path
-                    return Paths.get(uri.getPath()).toUri();
+        if (File.separator.equals("\\") && wildFlyConfig.contains("\\")) { // we are on the windows and we have windows absolute path
```

Comment:
It is not clear to me why we are sure we handle absolute path.

Reply:
@mchoma there is a test TestSimpleConfiguration.testPropertyUrl. You can debug it easily. Generally, both conditions are on windows true and next two lines will generate proper URI

Reply:
I mean these conditions are true also if relative path is used -Dwildfly.config.url=relative\path\to\wildfly-config.xml.

Probably just comment is misleading.

Reply:
Comment updated